<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTagDialog</class>
 <widget class="QDialog" name="NewTagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Properties:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QLineEdit" name="propertiesLineEdit"/>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Content:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QTextEdit" name="contentTextEdit"/>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
